course_info/templates/course_info/partials/people.html

```diff
@@ -10,200 +10,247 @@
-                <div ng-repeat="membership in warning.memberships"
-                     style="border-style: solid; border-width: 1px 0px 1px 0px; border-color: #ccc; padding:.5em; margin-top:1em;">
-                  {{ getProfileFullName(membership.profile) }}
-                  <badge role="{{ membership.profile.role_type_cd }}"></badge> {{ membership.user_id }}
-                  <strong>Enrolled as {{ membership.role.role_name }}</strong>
-                </div>
-              </div>
+          <div class="col-md-4">
+            <button type="button" class="btn btn-default dropdown-toggle pull-right"
+                                  data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              <i class="fa fa-info-circle"></i> Privacy Disclaimer <span class="caret"></span>
+            </button>
+            <ul class="dropdown-menu pull-right" style="width:30em; padding:1em;">
+              <li>The University has a legal responsibility to protect the privacy of students and the confidentiality of student records. Adding a member to the teaching staff of this course will allow them to view both "directory information" (i.e. student's name) and "non-directory information" (i.e. student's Harvard University ID#) for the students included in this course. The course list may include students who have asked that we not release any information about them, including the fact that they attend classes at Harvard. By adding a person to the teaching staff you are granting them access to view this information; others are not permitted to view these records. For these reasons, all members of the teaching staff should be reminded to keep this information confidential. If you have any questions, please contact the Office of the Registrar.</li>
+            </ul>
+          </div>
+        </div>
 
-              <!-- search term didn't find a user -->
-              <div ng-if="warning.type == 'notFound'">
-                <p>
-                <i class="fa fa-warning"></i>
-                {{ warning.searchTerm }} was not found. Please check the email or Harvard University ID and try again.
-                </p>
-              </div>
+        <div class="col-md-12" style="margin-bottom: 1em;">
+          <a href="https://connections.harvard.edu" target="_blank">
+            Go to the Harvard Directory
+          </a>
+          to find someone's official email address.
+        </div>
 
-              <div ng-if="warning.type == 'addFailed'">
-                <p><strong>{{ warning.searchTerm }}</strong> could not be added to this course at this time.</p>
-                <p>Please try again.</p>
+        <!-- begin add warning alerts -->
+        <uib-alert ng-repeat="warning in addWarnings" type="warning"
+                   close="closeAlert('addWarnings', $index)" class="col-md-12">
+          <div ng-switch on="warning.type">
```

Comment:
Eh, it's not big enough trigger my personal "yank it out" instinct, and it makes it clear from looking at a single file the different alerts that are possible.  I can more easily see the argument for making each type of alert its own partial, and just `ng-include`ing them from the switch statement here.  Not opposed to pulling things out, though, if you feel strongly about it.